.pipelines/templates/PackageBuild.yml

```diff
@@ -226,6 +231,12 @@ steps:
         max_cascading_rebuilds_arg="MAX_CASCADING_REBUILDS=${{ parameters.maxCascadingRebuilds }}"
       fi
 
+      additional_args=""
+      if [[ ${{ parameters.dailyBuildID }} == "lkg" ]]; then
+        echo "##[debug]Building toolchain for latest daily build without using daily build manifests"
+        additional_args="DAILY_BUILD_ID=${{ parameters.dailyBuildID }} DAILY_BUILD_ID_UPDATE_MANIFESTS=n"
+      fi
```

Comment:
The `DAILY_BUILD_ID` argument is separate from `DAILY_BUILD_ID_UPDATE_MANIFEST`. Don't bundle them together. This is a generic template for building packages - it should have no assumptions about how it's used.